.github/workflows/ci.yml

```diff
@@ -0,0 +1,238 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: Airflow
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+env:
+  BUILD_ID: ${{github.sha }}
+  MOUNT_LOCAL_SOURCES: "false"
+  MOUNT_HOST_AIRFLOW_VOLUME: "true"
+  FORCE_ANSWER_TO_QUESTIONS: "yes"
+  SKIP_CHECK_REMOTE_IMAGE: "true"
+  DB_RESET: "true"
+  VERBOSE: "true"
+  CI: "true"
+  # Should be a target branch
+  TRAVIS_BRANCH: "master"
+  TRAVIS: "true"
+jobs:
+  statics:
+    name: Static checks
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.6
```

Comment:
Is it possible to extract python version to variable ?  when we cherry-pick to 1.10 we use 3.5 as base version and it would be great to be able to change it in one place only. It was not possible in Travis, but maybe we can do it in GA? Maybe a global DEFAULT_PYTHON_VERSION env will do the job?